### ISSUE TITLE
[CAIA A11y Research - PILOT]: Authenticated Experience, Authenticated Menu - PILOT SESSION (January 2024)

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#74259]
# CAIA A11y Support

This is a child ticket of the research ticket [73875](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73875) - created to track work completed for the pilot session.

## Assignee
- @eli-oat 
- @EvanAtCoforma 
- @sara-amanda 

## Description

A CAIA a11y will participate as a user in a pilot of the [Authenticated Experience, Authenticated Menu's research study](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73875).

- **Researcher**: The VFS team researcher will test their prototype and conversation guide by conducting the pilot session with a CAIA A11Y.
- **User**: The CAIA A11Y team member will act as a real user in the pilot session.
- **Assistive Tech**: The CAIA A11Y team member will follow the prompts given by the VFS Researcher, while using and demonstrating how the behavior of a `screen reader` on `desktop` would behave.

### Assistive Tech Options

- **Available AT**: `Screen readers` on a `desktop` device is currently the primary source of AT that CAIA A11ys are able to support, in addition to `screen magnification` for user research sessions.
- **Additional Specifications:** If there is a particular `test user`, `browser` or `operating system` you would like us to use, please let us know in advance via this ticket's comments, tagging the assignee - or reach out to @coforma-terry in the CAIA Slack Channel: [#sitewide-content-accessibility-ia ](https://dsva.slack.com/archives/C01K37HRUAH)

## Team Info

- Team name: Authenticated Experience
- OCTO-DE product owner: Samara Strauss
- Product manager: Travis Chaill
- Designer: Liz Lantz
- FE engineer: Adam Whitlock
- Product/team Slack channel: #accountexp-authexp

## Pilot Session

- **CONFIRMED**: Friday, Jan. 26, 2024 at 11:30 a.m. ET
    - [ Invite sent to CAIA a11ys 1/23/2024, along with Slack thread information on the request and study](https://oddball.slack.com/archives/C05LB36ATAQ/p1706029489885569)
    - Eli will be representing CAIA a11ys
- **Prototype**: [CodePen](https://cdpn.io/pen/debug/JjzNWbR)
- **Research Convo Guide**: [Auth Exp. Auth MenuConvo Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/discovery-research/authenticated-menu-usability/conversation-guide.md)
- **Related Study:** _Per Liz, this is supporting the standardized header/footer work. https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14078 if that's helpful._

#### Mural Note Taking

If Evan is able to attend as an observer, perhaps he can take notes? 

> - [LL 1/24/2024](https://dsva.slack.com/archives/C909ZG2BB/p1706116331483779?thread_ts=1706116148.817899&cid=C909ZG2BB) Slightly new approach in Mural for note taking.  [Here’s the section setup for the pilot](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1706115685060/fe256c047a321ae25a40e7cd742c03e6b286738a?wid=0-1656387114105), loosely organized by the tasks in the [conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/discovery-research/authenticated-menu-usability/conversation-guide.md).  
> - **A couple of notes**: 
>     - When you’re typing in a sticky in Mural, if you press tab it’ll automagically create a new sticky right next to the one you were just in, using the same color. I think this makes it faster to keep up with the speed of convo if necessary, so I didn’t set up a ton of blank stickies this time. 
>     - If there are general observations or something that comes up that isn’t totally related to the task, it’s helpful to me if you leave the sticky in that section rather than try to separate it and organize it elsewhere.

## Definition of Done

```[tasklist]
### A11y Tasks
- [x] Add to a11y Capacity Calendar
- [x] Schedule CAIA a11y:`Eli`
- [x] Determine date and time: 1/26/24, 11:30 a.m. `CONFIRMED`
- [x] Review research [conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/discovery-research/authenticated-menu-usability/conversation-guide.md)
- [x] Review and [prototype in CodePen](https://cdpn.io/pen/debug/JjzNWbR)
- [x] Run pilot with VFS Researcher (see description section of this ticket for details).
- [x] [Review a Recent Example by Jamie](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/Deliverables/MHV%20Cartography%20Team/2024-01-18-Design-Review-Cartography-Team-New-Navigation.md) re: Findings
- [x] Share findings: Folder for Authenticated Experience in GH Deliverables for CAIA A11Ys: [teams/CAIA/accessibility/Deliverables/Authenticated Experience](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/Deliverables/Authenticated%20Experience/2024%2001%2026%20Authenticated%20Experience%20-%20Authenticated%20Menu%20-%20Pilot%20Research%20Session.md)
- [x] **Evan Note taking (optional)** ^^ Notes in Deliverables in GH folder linked above.
```

**Comment by sara-amanda** [comment]
@andaleliz - here is where we will track the pilot session. Invite sent to all CAIA a11ys, and will keep you posted. 😄 This is also linked in our CAIA a11y task list in your research ticket.

**Comment by sara-amanda** [comment]
@andaleliz @eli-oat @coforma-terry 

I went ahead and updated the `description section` of this pilot ticket to indicate the assistive tech we have available to test with. If there are any other specifications we should be made aware of, such as browser preference or even test user information, please let us know. Thanks so much! 👍

**Comment by sara-amanda** [comment]
@EvanAtCoforma I added you to this ticket, now that you have GitHub 🥳 access, so you can add your feedback.

**Comment by sara-amanda** [comment]
## Closing Ticket
- [x] Pilot session completed on Friday, Jan. 26, 2024, by Eli and Evan. 
- [x] Notes from the pilot can be found in the `Authenticated Experience Folder` for `CAIA` in GitHub. [Notes path: /teams/CAIA/accessibility/Deliverables](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/Deliverables/Authenticated%20Experience/2024%2001%2026%20Authenticated%20Experience%20-%20Authenticated%20Menu%20-%20Pilot%20Research%20Session.md)
- [x] Closing out this pilot ticket; all task list items completed.

cc: @coforma-terry
